sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Should we consider renaming the table name here from `adjust_derived_v1` to something a bit more representative of the content like `adjust_app_kpi_v1` or `adjust_app_overview_v1` (these are just example, just something that describes the contents well)?

Reply:
That is a great question. I am leaning more toward 'adjust_derived' because it's keeping with the pattern that Mozilla has established. Also, the downloaded data is pre-aggregated, so it feels derived to me. And it's called KPI based on Adjust's definition, not Mozilla's.